desktop/components/main_layout/templates/scripts.jade

```diff
@@ -115,15 +115,14 @@ if sharify
   != sharify.script()
 
 //- Add Google's jQuery
-script( src="//cdnjs.cloudflare.com/ajax/libs/jquery/2.1.4/jquery.min.js" )
+//- script( src="//cdnjs.cloudflare.com/ajax/libs/jquery/2.1.4/jquery.min.js" )
```

Comment:
@craigspaeth - any idea why this is here? We're bundling jQuery into `common.js` below

Reply:
Could be irrelevant now, but pre-jQuery supporting common js we mixed CDN dependencies with modules. I believe there's plenty of code around Force that still depends on a global `$` object, so if we remove this let's make sure to expose it via `window.$ = window.jQuery = require('jquery')` around [here](https://github.com/artsy/force/blob/master/desktop/lib/global_client_setup.coffee#L67).

Reply:
This bit here should take care of that: https://github.com/artsy/force/pull/1785/files#diff-11e9f7f953edc64ba14b0cc350ae7b9dR86

Reply:
Nice, works for me !